cmd/kwil-cli/cmds/database/call.go

```diff
@@ -104,51 +105,183 @@ func callCmd() *cobra.Command {
 
 // buildProcedureInputs will build the inputs for either
 // an action or procedure executon/call.
-func buildExecutionInputs(ctx context.Context, client clientType.Client, dbid string, proc string, inputs []map[string]any) ([][]any, error) {
+func buildExecutionInputs(ctx context.Context, client clientType.Client, dbid string, proc string, inputs []map[string]string) ([][]any, error) {
 	schema, err := client.GetSchema(ctx, dbid)
 	if err != nil {
 		return nil, fmt.Errorf("error getting schema: %w", err)
 	}
 
 	for _, a := range schema.Actions {
 		if strings.EqualFold(a.Name, proc) {
-			return buildActionInputs(a, inputs), nil
+			return buildActionInputs(a, inputs)
 		}
 	}
 
 	for _, p := range schema.Procedures {
 		if strings.EqualFold(p.Name, proc) {
-			return buildProcedureInputs(p, inputs), nil
+			return buildProcedureInputs(p, inputs)
 		}
 	}
 
 	return nil, fmt.Errorf("procedure/action not found")
 }
 
-func buildActionInputs(a *types.Action, inputs []map[string]any) [][]any {
+func decodeManyB64(inputs []string) ([][]byte, bool) {
+	b64Arr := [][]byte{}
+	b64Ok := true
+	for _, s := range inputs {
+		// in the CLI, if data has suffix ;b64, it is base64 encoded
+		if strings.HasSuffix(s, ";b64") {
```

Comment:
surprisingly shell doesn't treat what's following ; as a separate cmd on you machine? I just tried, it's treated as a separated cmd , `zsh: command not found: 6b4`

Reply:
Yeah I have to wrap it in quotes. I'm open to changing it to something else (like SQL's typecast) so that it doesn't need to be wrapped.

Reply:
Yeah I'm down for `::`, it's kind of consistent.  
 `;` is special in shell, generally.

Reply:
I guess would that get confusing with SQL? Since we would be doing `::base64`, which isn't really a type.

Reply:
For completeness: As per Jon's rec, I made it `<value_string>[b64]`

Reply:
That's a good point.  I don't have good suggestions on other symbols :/